README.md

```diff
@@ -42,7 +42,7 @@ This project is a collection of [PHP_CodeSniffer](https://github.com/squizlabs/P
 
 ### Requirements
 
-The WordPress Coding Standards require PHP 5.2 or higher and the [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) version **2.7.0** or higher.
+The WordPress Coding Standards require PHP 5.2 or higher and the [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) version **2.7.0** or higher, though it is strongly recommended to use PHPCS 2.8.1 or higher as older versions contain a security vulnerability.
```

Comment:
I'd suggest straight changing the wording to requiring 2.8.1. We don't want to be seen supporting older versions with known security vulnerabilities.